src/clp_logging/readers.py

```diff
@@ -556,7 +558,8 @@ def stream_ir_segment(self) -> Tuple[int, Optional[Metadata]]:
                 )
             self.istream.seek(self.offset)
 
-        bytes_read: int = self.readinto_buf(0)
+        bytes_read: int
```

Comment:
bytes_read is used inside the while loop coming. Since Python doesn't have separate scope for loops, this statement is to indicate that the variable is not just used in the place where it first gets initialized.